src/platform_impl/macos/app_delegate.rs

```diff
@@ -92,3 +101,17 @@ extern "C" fn application_will_terminate(_: &Object, _: Sel, _: id) {
   AppState::exit();
   trace!("Completed `applicationWillTerminate`");
 }
+
+extern "C" fn application_open_file(_: &Object, _: Sel, _: id, file: id) -> i8 {
```

Comment:
```suggestion
extern "C" fn application_open_file(_: &Object, _: Sel, _: id, file: id) -> bool {
```

Reply:
It seems should to be `BOOL`(`cocoa::base::BOOL`)